packages/story-editor/src/app/taxonomy/taxonomyProvider.js

```diff
@@ -161,11 +161,11 @@ function TaxonomyProvider(props) {
       }
       try {
         response = await getTaxonomyTerm(termsEndpoint, {
-          search: name,
+          search_items: name,
```

Comment:
https://github.com/google/web-stories-wp/blob/main/includes/Taxonomy/Tag_Taxonomy.php#L61

Reply:
Attention. You mixed up two completely different things here.

search_items is just the name of the label for when you want to display „Search Tags“ in the UI. See how it‘s in the „labels“ array in the code you linked to.

It‘s not related to the REST API in any way.

search ought to be correct for REST API querying, but consult the WP REST API reference documentation or ask us to confirm.

Reply:
yeah, i realize that now. Sorry about that - i must have gotten some wires crossed and thought this was the path forward. thought it was a bit odd but i was getting results i wanted it seemed. It's all fixed now. Appreciate the call out.